source/jormungandr/jormungandr/interfaces/v1/serializer/fields.py

```diff
@@ -0,0 +1,106 @@
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from __future__ import absolute_import, print_function, unicode_literals, division
+from flask import g
+from jormungandr.interfaces.v1.make_links import create_internal_link, create_external_link
+from jormungandr.interfaces.v1.serializer.base import EnumField, PbNestedSerializer, EnumListField
+from jormungandr.interfaces.v1.serializer.time import LocalTimeField, DateTimeField, PeriodSerializer
+import operator
+import serpy
+
+class MultiLineStringField(serpy.Field):
+
+    def to_value(self, value):
+        if hasattr(g, 'disable_geojson') and g.disable_geojson:
+            return None
+
+        lines = []
+        for l in value.lines:
+            lines.append([[c.lon, c.lat] for c in l.coordinates])
```

Comment:
except that it doesn't work, I get this error: `OverflowError: Maximum recursion level reached`